package/setup.py

```diff
@@ -515,14 +516,31 @@ def dynamic_author_list():
         print(template.format(author_string), file=outfile)
 
 
+def long_description(readme):
+    """Create reST SUMMARY file for PyPi."""
+
+    with open(abspath(readme)) as summary:
+        buffer = summary.read()
+    # remove top heading that messes up pypi display
+    m = re.search('====*\n[^\n]*README[^\n]*\n=====*\n', buffer,
+                  flags=re.DOTALL)
+    assert m, "README.rst does not contain a level-1 heading"
+    return buffer[m.end():]
+
```

Comment:
Shouldn't there be two blank lines here? I would have expected pep8speaks to pick this one up, so maybe there's a PEP8 special case for empty lines between functions and `if __name__ == '__main__'` that I'm not aware of.

Reply:
fixed (together with other PEP8 complaints)